_includes/v21.2/known-limitations/restore-multiregion-match.md

```diff
@@ -0,0 +1,50 @@
+[`REGIONAL BY TABLE`](multiregion-overview.html#regional-tables) tables can be restored **only** if the [table locality](multiregion-overview.html#table-locality) matches that of the target database.
+
+    The following must also be true for `RESTORE` to be successful:
+
+    * The regions of the source database and the regions of the destination database have the same set of regions.
+    * The regions were added to each of the databases in the same order.
+    * The databases have the same primary region.
```

Comment:
'primary region' could link to https://www.cockroachlabs.com/docs/v21.2/set-primary-region